platform-dom/src/main/java/com/softicar/platform/dom/input/IDomDisableable.java

```diff
@@ -7,7 +7,7 @@
  *
  * @author Oliver Richers
  */
-public interface IDomEnableable extends IDomElement {
+public interface IDomDisableable extends IDomElement {
```

Comment:
Renamed this interface. The name is still awkward, but at least it better describes what this is all about, i.e. DOM elements that can be **disabled**.